.gitlab-ci.yml

```diff
@@ -26,6 +26,7 @@ stages:
     - me_dev
     - top_dev
     - sw_dev
+    - accelerator_fix
```

Comment:
After CI finishes running to confirm changes are good, remove this before merge.